.rubocop.yml

```diff
@@ -791,6 +791,7 @@ Style/PredicateName:
   # helpers in the form of `have_something` or `be_something`.
   Exclude:
     - '**/spec/**/*'
+    - '**/test/**/*'
```

Comment:
Ah, wait. I didn't check that, so I'm not so sure about renaming (thanks for pointing this out). Then I'd leave this change as it is, but let's ask the others...

@josepjaume @beagleknight @oriolgual any input, guys?